conda/meta.yaml

```diff
@@ -11,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   noarch: python
-  entry_points:
-    - zppy = zppy.__main__:main
 
 requirements:
   host:
-    - python >=3.9
+    - python {{ python_min }}
```

Comment:
Here and below, conda-forge has moved to defining `python_min` (`3.9` by default) and ensuring that the package bets built and tested with this minimum version.  The idea is that thing tend to break more often when we only test with the newest python than when we only test with the oldest.  See https://github.com/conda-forge/cfep/blob/main/cfep-25.md for details.